week1/data_processor.py

```
@@ -0,0 +1,48 @@
+import pandas as pd
+import yaml
```

Ruff

`yaml` imported but unused

Remove unused import: `yaml`

(F401)